tasks/main.yml

```diff
@@ -1,2 +1,8 @@
 ---
-# tasks file for skeleton
+- name: Download installer
+  become: true
+  get_url:
```

Comment:
We prefer the fully-qualified collection names (FQCNs) to avoid warnings from the Ansible linter.  In this case the FQCN is `ansible.builtin.get_url`.

Reply:
I amended [087d664](https://github.com/cisagov/ansible-role-metasploit/pull/5/commits/087d664f8a70cbe560f094b98ee47217b3118122) to use the FQCN.